transactions.go

```diff
@@ -74,6 +74,17 @@ func (t *Transactions) BatchAdd(block BlockID, transactions []Transaction, verif
 	}
 }
 
+// BatchBufferAdd adds transactions to buffer without adding them to index
+// used on node's start
+func (t *Transactions) BatchBufferAdd(txs []*Transaction) {
+	t.Lock()
+	defer t.Unlock()
+
```

Comment:
Care to rename to `BatchUnsafeAdd`? (adds transactions to the index without any validation).

Reply:
Why unsafe though?

Reply:
Since it adds transactions directly to the buffer without any validation (unlike `BatchAdd`).